rust/Earthfile

```diff
@@ -1,96 +1,81 @@
-VERSION --global-cache 0.7
+VERSION --use-function-keyword --global-cache 0.7
```

Comment:
--use-function-keyword only available in version 0.8 repo and samples pinned to version 0.7

Reply:
Hi Alex, that feature flag is available since 0.7.22 https://github.com/earthly/earthly/blob/v0.7.22/features/features.go#L67
Maybe your CLI is not updated?